azure-ml-deployment/dai_deploy.yml

```diff
@@ -0,0 +1,9 @@
+name: dai_deploy
+dependencies:
+  - python=3.6.2
+  - pip:
+    - azureml-defaults
+    - numpy
+    - pandas
+    - https://h2ostore12792.blob.core.windows.net/mojodependencies/daimojo-2.0.1%2Bmaster.478-cp36-cp36m-linux_x86_64.whl
```

Comment:
these should probably point to the released artifacts in s3 imho

Reply:
I'm going to remove this file since it gets created every time the script is run. 
The script takes the path for the daimojo and datatable packages and upload them to a blob storage under the user subscription.